example/ConquerClub.UnitTests/Game_specs.cs

```diff
@@ -0,0 +1,87 @@
+﻿using ConquerClub.Domain;
+using ConquerClub.Domain.Events;
+using NUnit.Framework;
+using Qowaiv.Identifiers;
+using Qowaiv.Validation.Abstractions;
+using Qowaiv.Validation.TestTools;
+using System.Linq;
+using static ConquerClub.UnitTests.Arrange;
+using Commands = ConquerClub.Domain.Commands;
+
+namespace Game_specs
+{
+    public class Start_Game
+    {
+        [Test]
+        public void With_two_players_includes_third_neutral_player()
+        {
+            var command = new Commands.Start(
+                Game: GameId,
+                Players: 2,
+                RoundLimit: 10,
+                Continents: new[]
+                {
+                    new Commands.Continent(
+                        Name: "Benelux",
+                        Bonus: 3,
+                        Territories: new []
+                        {
+                            Netherlands,
+                            Belgium,
+                            Luxembourg,
+                        }),
+                },
+                Countries: new[]
+                {
+                    new Commands.Country("Netherlands", new []{ Belgium }),
+                    new Commands.Country("Belgium", new []{ Netherlands, Luxembourg }),
+                    new Commands.Country("Luxembourg", new []{ Belgium }),
+                });
+
+            var result = Handle(command);
+
+            ValidationMessageAssert.IsValid(result);
+
+            CollectionAssert.AreEquivalent(new[]
+            {
+                Player.P1,
+                Player.P2,
+                Player.Neutral
+            },
+            result.Value.Countries.Select(c => c.Owner).Distinct());
+        }
+    }
+
+    public class Deploy
+    {
+        [Test]
+        public void Can_only_be_applied_the_active_player()
```

Comment:
Only one side of the "_only_" is tested. The case in which P1 deploys is not tested.